docs/styles.css

```diff
@@ -1,10 +1,31 @@
+@font-face {
+	font-family: 'open-sans';
+	src: url('fonts/OpenSans-VariableFont_wdth,wght.ttf') format('truetype');
+	font-weight: 400;
+	font-style: normal;
+}
+
 :root {
 	--scalar-custom-header-height: 50px;
+	--scalar-font: 'open-sans', 'sans-serif';
+    --scalar-font-code: 'open-sans', 'sans-serif';
+	--scalar-radius: 3px;
+	--scalar-radius-lg: 8px;
+    --scalar-radius-xl: 12px;
+	--scalar-micro: 13px;
+	--scalar-mini: 15px;
+	--scalar-small: 15px;
+	--markdown-line-height: 1.65;
 }
 
+/* Header styling */
+
+.custom-header .img {min-width: 180px !important; height: auto;}
+.dark-mode .custom-header .img {source:"https://app.theneo.io/opusdns/api/rest-api-documentation/events"}
```

Comment:
I'm not sure what this line is supposed to be doing. We're referencing an URL on `theneo`, which we absolutely should not be doing. It also makes it seem like it's supposed to be an image source, but it's not?